.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "daily"
+    time: "11:00"
+  open-pull-requests-limit: 10
```

Comment:
```suggestion
  open-pull-requests-limit: 5
```

Reply:
And this one? 5 seems like a great place to start. Like you said, easily reversible so we can increase later if needed.